src/apis/study-room/study-step.ts

```diff
@@ -0,0 +1,7 @@
+import { http } from '../api';
+
+type Params = { studyId: number; step: string };
+export const studyStep = async ({ studyId }: Params): Promise<void> => {
```

Comment:
사소하지만 요런 개행도 챙겨주면 좋을 것 같습니다~
```suggestion
type Params = { studyId: number; step: string };

export const studyStep = async ({ studyId }: Params): Promise<void> => {
```